uefi_app/SbsaAvsMain.c

```diff
@@ -477,14 +477,16 @@ ShellAppMainsbsa (
   Print(L"\n      *** Starting IO Virtualization tests ***  \n");
   Status |= val_smmu_execute_tests(g_sbsa_level, val_pe_get_num());
 
-  /*
-   * Configure Gic Redistributor and ITS to support
-   * Generation of LPIs.
-  */
-  configureGicIts();
-
-  Print(L"\n      *** Starting PCIe Exerciser tests ***  \n");
-  Status |= val_exerciser_execute_tests(g_sbsa_level);
+  if (g_sbsa_level > 3) {
+    /*
+    * Configure Gic Redistributor and ITS to support
+    * Generation of LPIs.
+    */
+    configureGicIts();
```

Comment:
I would think configureGicIts has no relationship with the SBSA levels. So, it should be moved outside of the 'if'.

Reply:
I think that it is done for PCIe exerciser tests. As at this point we have only them and NIST tests run.

Reply:
Yes, but that is an artificial restriction based on the tests we have today. It is not dictated by the specification levels. so, would like to keep them separate.